### PR TITLE
Update zh-hant translation for revised Quick-Facts

### PR DESCRIPTION
**Pull request by Lafudoci**
Base on PR #99 and some markdown format revisions.

### CODE REVIEW

File: monero-outreach-docs/translations/zh-hant/quickfacts/quickfacts_zh-hant.md [hunk 1]
```diff
@@ -57,47 +53,47 @@ Monero 一詞是取自世界語(Esperanto)。之所以選擇使用世界語是
 
 **mon-:** 金錢
 **-er-:** 最小單位
- **-o :** 一個事物 (代表一個名詞)
+**-o :** 一個事物 (代表一個名詞)
```
Comment (lh1008) [inline monero-outreach-docs/translations/zh-hant/quickfacts/quickfacts_zh-hant.md]: We should add some spaces between lines so they can show up separately. Let me show you: 

**mon-:** 金錢

**-er-:** 最小單位

**-o :** 一個事物 (代表一個名詞)

If we don't add the spaces this section will appear in one line only.

File: monero-outreach-docs/translations/zh-hant/quickfacts/quickfacts_zh-hant.md [hunk 2]
```diff
@@ -57,47 +53,47 @@ Monero 一詞是取自世界語(Esperanto)。之所以選擇使用世界語是
...
++ **平均每小時交易數：** 149 (天平均)
 
-+ **全網挖礦CPU核心數：** 14,499,178
++ **全網挖礦CPU核心數：** 22,234,297
 
-+ **貨幣流通量：** 16,570,126 XMR (估計值)
++ **貨幣流通量：** 16,758.217 XMR (估計值)
 
-+ **貨幣市值：** $1,503,928,296 USD (約佔0.77%加密貨幣總市值)
++ **貨幣市值：** $743,944,945 USD (約佔0.77%加密貨幣總市值)
```
Comment (lh1008) [inline monero-outreach-docs/translations/zh-hant/quickfacts/quickfacts_zh-hant.md]: We found the information we had posted was incorrect and thanks to @el00ruobuob we were able to fix it. This is the string with the correct information.

Market Capitalization: $732,932,071 USD (~0.65% of total cryptocurrency market cap)

File: monero-outreach-docs/translations/zh-hant/quickfacts/quickfacts_zh-hant.md [hunk 3]
```diff
@@ -109,13 +105,11 @@ _Monero 的區塊獎勵會緩慢下降直到 2022 年五月，屆時將有 一
...
 
-防彈協定：[這項開發](https://getmonero.org/2017/12/07/Monero-Compatible-Bulletproofs.html)可以讓交易在區塊鏈上儲存的效率提升，並帶來更快的交易速度。
-
-**Kovri：**這會是一個隱私保護功能的升級，為 Monero 交易加上第四層保護。 Kovri 使用**[garlic-encryption](https://getmonero.org/resources/moneropedia/garlic-encryption.html)**(就像是 Tor on steroids) 用來遮蔽發送者與接收者在網路上的 IP 位址。如此一來，在 Monero 上的交易每一層面包括發送者、接收者、交易金額和交易本身(使用 Kovri)將都會得更有隱私且更加安全。
+**Kovri：** 這會是一個隱私保護功能的升級，為 Monero 交易加上第四層保護。 Kovri 使用 **[garlic-encryption](https://getmonero.org/resources/moneropedia/garlic-encryption.html)** (就像是 Tor on steroids) 用來遮蔽發送者與接收者在網路上的 IP 位址。如此一來，在 Monero 上的交易每一層面包括發送者、接收者、交易金額和交易本身(使用 Kovri)將都會得更有隱私且更加安全。
 
-**硬體錢包：**Trezor Model T 硬體錢包已經將 Monero 列入**[支援清單](https://trezor.io/coins/#XMR)**，而**[Ledger 硬體錢包](https://support.ledgerwallet.com/hc/en-us/articles/360006352934-Monero-XMR-Advanced)**已可以在 Monero 的官方 GUI 錢包中使用。而另外 Monero社群更成功募資支持了一個團隊以**[打造 Kastelo 開源硬體錢包](http://kastelo.org/)**，系統將基於 Monero ethos 上開發。這些硬體錢包都完全支援 Monero 的交易。
+**硬體錢包：** 熱門的 Ledger 和 Trezor 硬體錢包都正在加入 Monero（Ledger 已可以在 Monero 官方的 GUI v0.12.3 和 CLI v0.12.2 錢包中使用），而另外 Monero社群更成功募資支持了一個團隊以 **[打造 Kastelo 開源硬體錢包](http://kastelo.org/)** ，系統將基於 Monero ethos 上開發。這些硬體錢包都完全支援 Monero 的交易。
```
Comment (lh1008) [inline monero-outreach-docs/translations/zh-hant/quickfacts/quickfacts_zh-hant.md]: We also made an addition to the last sentence of this paragraph. Added some missing words and made it a little more understandable. This is the last sentence:

"Both Ledger and Trezor hardware wallet options were anticipated to be operational by the end of 2018."

You can take a look at #99 and see the change in the last sentence of the paragraph.

**Comment by lh1008** [comment]
Approved the date change.

File: monero-outreach-docs/translations/zh-hant/quickfacts/quickfacts_zh-hant.md (hunk 1 above)
Comment (Lafudoci) [reply monero-outreach-docs/translations/zh-hant/quickfacts/quickfacts_zh-hant.md]: Revised.

File: monero-outreach-docs/translations/zh-hant/quickfacts/quickfacts_zh-hant.md (hunk 2 above)
Comment (Lafudoci) [reply monero-outreach-docs/translations/zh-hant/quickfacts/quickfacts_zh-hant.md]: Revised.

File: monero-outreach-docs/translations/zh-hant/quickfacts/quickfacts_zh-hant.md (hunk 3 above)
Comment (Lafudoci) [reply monero-outreach-docs/translations/zh-hant/quickfacts/quickfacts_zh-hant.md]: Revised.

**Comment by Lafudoci** [comment]
OK, I believe it's ready!

**Pull request merged**